spec/index.bs

```diff
@@ -6543,6 +6547,29 @@ following members:
         </div>
 </dl>
 
+<div algorithm data-timeline=device>
+    <dfn abstract-op>copy of buffer binding</dfn>(|resource|)
```

Comment:
There's no need for this to copy the object, and calling it a "copy" can be confusing because it could sound like it copies the entire buffer. I would maybe just call it `get buffer binding` and remove the copy on 6568.

Reply:
Done